parsing/parse.ml

```diff
@@ -121,7 +121,14 @@ and core_type = wrap_menhir Parser.Incremental.parse_core_type
 and expression = wrap_menhir Parser.Incremental.parse_expression
 and pattern = wrap_menhir Parser.Incremental.parse_pattern
 
-
+module Longident = struct
+  let any = wrap_menhir Parser.Incremental.parse_any_longident
+  let value = wrap_menhir Parser.Incremental.parse_val_longident
+  let constr = wrap_menhir Parser.Incremental.parse_constr_longident
+  let mod_ext = wrap_menhir Parser.Incremental.parse_mod_ext_longident
+  let simple_module = wrap_menhir Parser.Incremental.parse_mod_longident
+  let typ = wrap_menhir Parser.Incremental.parse_mty_longident
+end
```

Comment:
Nitpick: I'm not fond of `mod_ext` and `simple_module`. The mod/module discrepancy is not nice, and I would like to see `path` in the name, because I think of a simple module *path*, not a simple *module*. 

If it was me, I would do away with the Longident module and expose

```ocaml
val_ident
constr_ident
typ_ident
simple_module_path
{ext,full}_module_path (* extended_ is too long *)
```

Reply:
The name "extended module path" is already used in the manual (to describe which kind of modules are accepted in open in signature).

Reply:
I applied the change, mainly for the `Longident.any` ⇒ `longident` renaming.